packages/start/src/client/Meta.tsx

```diff
@@ -150,7 +150,10 @@ export const useMetaElements = () => {
           //             let controller
           //             ex.value = new ReadableStream({
           //               start(c) {
-          //                 controller = c
+          //                 controller = {
+          //                   enqueue: chunk => { try { c.enqueue(chunk); } catch {} },
+          //                   close: () => { try { c.close(); } catch {} },
+          //                 }
```

Comment:
Doing this here seems to be the place that is the most defensive - anywhere "further out", it could swallow errors e.g. if `controller` is still undefined etc.